app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.js

```diff
@@ -74,10 +73,112 @@ class SubjectAreaDropdown extends React.Component {
       hasReachedMultiselectLimit: false,
     }
 
+    const chooseBorderColorFromProps = (
+      theme,
+      validationStatus = 'default',
+      isFocused,
+    ) =>
+      ({
+        error: theme.colorError,
+        success: theme.colorSuccess,
+        warning: theme.colorWarning,
+        default: isFocused ? '#2684FF' : theme.colorBorder,
+      }[validationStatus])
+
+    const gridUnitValue = parseInt(this.props.theme.gridUnit, 10)
```

Comment:
@tamlyn Does this make sense to have? I wasn't sure if I was understanding this correctly, but using `calc` inside `customReactSelectStyles` wasn't working, so all my calculations are done like this below: `${gridUnitValue / 2}px`

Reply:
i think you could import `css` from emotion and  do 
```
padding: css`calc(stuff)`
```
if you wanted

Reply:
I think this is fine. Integrating with third party components is necessarily a bit messy.

Reply:
```
Interpolating functions in css calls is deprecated and will be removed in the next major version of Emotion.
If you want to have a css call based on props, create a function that returns a css call like this
let dynamicStyle = (props) => css`color: ${props.color}`
It can be called directly with props or interpolated in a styled call like this
let SomeComponent = styled('div')`${dynamicStyle}`
```

I reckon adding 
```
(props) => css`calc(stuff)`
```
into this already beasty object is overkill